src/app/models/form.server.model.ts

```diff
@@ -163,10 +167,13 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     {
       title: {
         type: String,
-        validate: [
-          /^[a-zA-Z0-9_\-./() &`;'"]*$/,
```

Comment:
/justMusing: Ooohh, so we did allow some special characters even previously, we could have set the prefix as `(Template) `  and reduce the changeset 🤔